src/fragments/lib-v1/auth/android/signin/20_confirmSignUp.mdx

```diff
@@ -57,3 +57,9 @@ RxAmplify.Auth.confirmSignUp("username", "the code you received via email")
 
 </Block>
 </BlockSwitcher>
+
+You will know the sign up flow is complete if you see the following in your console window:
```

Comment:
Note: this was added here (and under /ios) because it was removed from /native_common as it is not applicable to flutter.